app/src/lib/date-utils.ts

```diff
@@ -36,3 +36,20 @@ export function getTodayUtcDate(): string {
 
   return todayBucket
 }
+
+export function getHourFromUtcDate(date: string): string {
+  dayjs.extend(dayJsutcPlugin)
+
+  const time = dayjs(date)
+
+  return time.format('HH:mm')
+}
+
+// Returns today's date in the format of `DD/MM/YYYY`
+export function getTodayStringTime(): string {
```

Comment:
I wouldn't add `String` to the function name—it's generally a bad practice to put types in function names especially when their return type is inferable through intelliSense. I'd call this `getTodayISODate`